massgov/mayflower
# DP-22974 KeyMessage overlay visual regression in openmass develop

Any PRs being created needs a changelog.txt file before being merged into dev. See: [Change Log Instructions](https://github.com/massgov/mayflower/blob/develop/docs/for-developers/changelog-instructions.md)


## Description

It adjusts the key message component to fix a regression issue where the image was showing above the text in some cases that shouldn't be like that.

## Related Issue / Ticket

- [JIRA issue](https://massgov.atlassian.net/browse/DP-22974)
- [Github issue]()

## Steps to Test


1. Open PatternLab, open the component "Key Message As Solid Image Primary" under "Organisms" > "By template"  and compare the mobile version with https://mayflower.digital.mass.gov/patternlab/v/11.13.0/index.html?p=organisms-key-message-as-solid-image-primary , both should look the same.
2. Open the remaining "key message" variations and compare with https://mayflower.digital.mass.gov/patternlab/v/11.13.0/index.html , both should look the same
3. Open "Marketing campaign" under "pages" and compare it with https://mayflower.digital.mass.gov/patternlab/?p=pages-marketing-campaign , validate that the key message components (top of the page and bottom of the page) on mobile matches.

## Screenshots
### Wrong: 
<img width="533" alt="j-r-capture 2021-09-17 a la(s) 18 41 55" src="https://user-images.githubusercontent.com/141685/133856899-e14c8f9f-df12-4410-a99c-60552d3adba8.png">
### Right:
<img width="499" alt="j-r-capture 2021-09-17 a la(s) 18 41 59" src="https://user-images.githubusercontent.com/141685/133856912-a6395b6f-6e02-4343-a193-d20a98db5371.png">


## Additional Notes:

Anything else to add?

#### Impacted Areas in Application


*

#### @TODO


*

#### Today I learned...
